src/app/public/src/modules/nav/nav.component.html

```diff
@@ -4,10 +4,10 @@
       *ngFor="let route of routes"
       [ngClass]="{'active': route.isActive, 'current': route.isCurrent}">
       <a class="stache-nav-anchor"
-        routerLink="{{route.path}}"
+        stacheRouterLink="{{route.path}}"
         fragment="{{route.fragment}}"
         queryParamsHandling="merge"
-        (click)="navigate(route)">{{route.name}}</a>
+        (click)="navigate($event, route)">{{route.name}}</a>
```

Comment:
This and the below instance are both necessary because we need to stop the component from navigating to the anchor on the initial page if we open in a new tab/window.

Reply:
Not sure I follow.  Removing the click event entirely from this component seems to work fine when opening in a new tab, or window, or clicking directly on the page.  Am I missing something?

Reply:
We definitely still need the event in the component, but it looks like removing it here still feeds the event to the function, if that's what you mean. I felt that leaving the event made the workflow more readable.